.github/workflows/anaconda_pkg_build.yml

```diff
@@ -69,3 +69,28 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           push: ${{ github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags') }}
+
+      - name: Docker meta for Concourse
+        id: concourse-meta
+        uses: docker/metadata-action@3a3bb3a81753dc99f090d24ee7e5343838b73a96
+        with:
+          images: |
+            continuumio/concourse-pkg-build
+            public.ecr.aws/y0o4y9o3/concourse-pkg-build
```

Comment:
Would it also help to push a single arch image with its own tag (e.g. $version-$suffix, like 2021-09-30-amd64) to continuumio/anaconda-pkg-build and public.ecr.aws/y0o4y9o3/anaconda-pkg-build?

Reply:
I think that would be ideal, but I did not want to cause weird conflicts between the two steps. Would that be something like?:

```
        with:
          images: |
            continuumio/anaconda-pkg-build
            public.ecr.aws/y0o4y9o3/anaconda-pkg-build
          tags: |
            type=ref,event=branch,suffix=amd64
            type=ref,event=pr,suffix=amd64
            type=match,pattern=pkg-build-(.*),group=1,suffix=amd64
```

Reply:
that should do it.